app/mm-redux/actions/users.ts

```diff
@@ -244,13 +235,6 @@ export function loadMe(): ActionFunc {
     };
 }
 
-export function getTotalUsersStats(): ActionFunc {
```

Comment:
This is not a blocking question.  I notice several functions are being removed....is it part of a house-keeping where we're removing un-used functions ?

Reply:
Cleanup